source/index.html.md

```diff
@@ -98,11 +99,32 @@ The endpoint accepts one parameter and a [Basic authentication](https://en.wikip
 
 `POST https://app.sixpark.com.au/oauth/token`
 
+### Scopes
+
+Name | Description
+--------- | -----------
+questionnaire.questions.read | Read questionnaire question data
+questionnaire.results.read | Read questionnaire result data
+users.read | Read user data
+users.write | Write user data
+users.accounts.read | Read user accounts data
+users.accounts.write | Write user accounts data
+users.accounts.results.read | Read user accounts result data
```

Comment:
I made a comment about this one in the other PR.

Reply:
oh yep that's right and I renamed it lol will fix